scripts/x-compile-android-x86.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
This looks close to `scripts/x-compile-android-amd64.sh` file. Can we do a common script with architecture argument?

Reply:
Yea, probably. But maybe next time - those scripts are more like - run and forget, maybe we will get rid of them completely some day.